FindGLEW.cmake

```diff
@@ -20,3 +20,13 @@ set(GLEW_INCLUDE_DIRS ${GLEW_INCLUDE_DIR})
 set(GLEW_LIBRARIES ${GLEW_LIBRARY})
 
 mark_as_advanced(GLEW_LIBRARY GLEW_INCLUDE_DIR)
+
+if(NOT ${CMAKE_VERSION} VERSION_LESS "3.0")
+    if(NOT TARGET GLEW::GLEW)
+        add_library(GLEW::GLEW INTERFACE IMPORTED)
+        set_target_properties(GLEW::GLEW PROPERTIES
+            INTERFACE_INCLUDE_DIRECTORIES "${GLEW_INCLUDE_DIRS}"
+            INTERFACE_LINK_LIBRARIES "${GLEW_LIBRARIES}"
```

Comment:
add `interface_compile_definitions` as well

Reply:
compile definitions are not available in the cmake file

Reply:
oh, this is unfortunate situation. then, let's merge as is...

Reply:
Anyways, working on a fix..

Reply:
Fixed (maybe).
Let's wait on ci: https://travis-ci.com/madebr/conan-glew/builds/102364088